tmt/steps/discover/__init__.py

```diff
@@ -158,7 +158,7 @@ def save(self) -> None:
 
         # Create tests.yaml with the full test data
         raw_test_data = {
-            test.name: test.export(format_=tmt.base.ExportFormat.DICT)
+            test.name: test._export()
```

Comment:
What about using `Test.export_collection()` directly in the `self.write()` command below?

Reply:
That would export a list of tests instead of a mapping, i.e. the context of a file is supposed to be

```yaml
/test/foo:
  name: foo
  duration: 1h
  ...
```

but we'd get

```yaml
- name: foo
  duration: 1h
  ...
```

Reply:
Ah, right, forget it's still a `dict`. I was living a bit in the future ;-)